api/lib/gateways/UHW/UHWFetchDocuments.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path');
+const { loadSQL } = require('../../Utils');
+const { Systems } = require('../../Constants');
+const { fetchCustomerDocumentsSQL } = loadSQL(path.join(__dirname, 'sql'));
+
+module.exports = options => {
+  const db = options.db;
+  const buildDocument = options.buildDocument;
+
+  const fetchCustomerDocumentsQuery = async id => {
+    return await db.request(fetchCustomerDocumentsSQL, [
+      { id: 'id', type: 'Int', value: id }
+    ]);
+  };
+
+  const processDocuments = results => {
+    return results.map(doc => {
+      return buildDocument({
+        id: doc.DocNo,
+        title: 'Document',
+        text: doc.DocDesc + `${doc.title ? ' - ' + doc.title : ''}`,
```

Comment:
mixing string concatenation and interpolation (let go all-out interpolation)

Reply:
I changed it bu I will revert it now. It seems that there was a reason for it (the conditional in the middle (?)): 

<img width="380" alt="image" src="https://user-images.githubusercontent.com/54268893/73848982-0b5d3580-4821-11ea-8879-128ecebe0975.png">

Reply:
Unless I misunderstood something

Reply:
I meant it should be `${doc.DocDesc}${doc.title ? ' - ' + doc.title : ''}`